frontend/src/app/Settings/Facility/operations.graphql

```diff
@@ -0,0 +1,43 @@
+query GetManagedFacilities {
```

Comment:
This is loading so much more data than needed for the replacement
`const GET_FACILITIES = gql
  query GetManagedFacilities {
    organization {
      testingFacility {
        id
        cliaNumber
        name
      }
    }
  }
;`